python/ray/autoscaler/aws/cloudwatch/example-cloudwatch-agent-config.json

```diff
@@ -0,0 +1,175 @@
+              "ray_node_disk_utilization_percentage": "Percent",
+              "ray_node_mem_available": "Bytes",
+              "ray_node_mem_total": "Bytes",
+              "ray_node_mem_used": "Bytes",
+              "ray_node_network_receive_speed": "Bytes",
+              "ray_node_network_received": "Bytes",
+              "ray_node_network_send_speed": "Bytes",
+              "ray_node_network_sent": "Bytes",
+              "ray_avg_num_executed_tasks": "Count",
+              "ray_avg_num_scheduled_tasks": "Count",
+              "ray_avg_num_spilled_back_tasks": "Count",
+              "ray_object_manager_num_pull_requests": "Count",
+              "ray_object_store_available_memory": "Bytes",
+              "ray_object_store_used_memory": "Bytes",
+              "ray_object_store_fallback_memory":"Bytes",
+              "ray_object_store_num_local_objects": "Count",
+              "ray_object_directory_subscriptions": "Count",
+              "ray_object_directory_added_locations": "Count",
+              "ray_object_directory_removed_locations": "Count",
+              "ray_object_directory_lookups": "Count",
+              "ray_object_directory_updates": "Count",
+              "ray_pending_actors": "Count",
+              "ray_pending_placement_groups": "Count",
+              "ray_raylet_cpu": "Count",
+              "ray_raylet_mem": "Bytes",
+              "ray_internal_num_spilled_tasks": "Count",
+              "ray_internal_num_infeasible_tasks": "Count",
+              "ray_internal_num_processes_started": "Count",
+              "ray_internal_num_received_tasks": "Count",
+              "ray_internal_num_dispatched_tasks": "Count",
+              "process_max_fds": "Count",
+              "process_open_fds": "Count",
+              "process_resident_memory_bytes": "Bytes",
+              "process_virtual_memory_bytes": "Bytes",
+              "process_start_time_seconds": "Seconds",
+              "process_cpu_seconds_total": "Seconds",
+              "autoscaler_config_validation_exceptions": "Count",
+              "autoscaler_node_launch_exceptions": "Count",
+              "autoscaler_pending_nodes": "Count",
+              "autoscaler_reset_exceptions": "Count",
+              "autoscaler_running_workers": "Count",
+              "autoscaler_started_nodes": "Count",
+              "autoscaler_stopped_nodes": "Count",
+              "autoscaler_update_loop_exceptions": "Count",
+              "autoscaler_worker_create_node_time": "Seconds",
+              "autoscaler_worker_update_time": "Seconds",
+              "autoscaler_updating_nodes": "Count",
+              "autoscaler_successful_updates": "Count",
+              "autoscaler_failed_updates": "Count",
+              "autoscaler_failed_create_nodes": "Count",
+              "autoscaler_recovering_nodes": "Count",
+              "autoscaler_successful_recoveries": "Count",
+              "autoscaler_failed_recoveries": "Count"
```

Comment:
Disclaimer: I'm not that familiar with Ray's current monitoring tools.

Is this going to be tricky to keep in sync with Ray monitoring code? What if we add or remove a metric somewhere?
cc @Zyiqin-Miranda @simon-mo @ckw017 @ijrsvt

Reply:
We can update this [line](https://github.com/ray-project/ray/blob/master/python/ray/autoscaler/_private/prom_metrics.py#L10) to stay in sync with this file as well. I'm not familiar enough with cloudwatch to know what the result of adding a new metric would be

Reply:
It would be great to keep in sync with this [line](https://github.com/ray-project/ray/blob/master/python/ray/autoscaler/_private/prom_metrics.py#L10) that includes all Prometheus metrics current ray is emitting.

The list of Prometheus metrics here is only for metrics unit. We basically ship all Premetheus format ray metrics to CloudWatch console (In L94-96 with “metric_selectors”:[” “]). Newly-added metrics will still be shipped to CloudWatch without updating the list here.